[Failing Test] /var/lib/docker/tmp/buildkit-mount2116837088/lib64: no such file or directory

### Which jobs are failing?

`build-master`

### Which tests are failing?

N/A

### Since when has it been failing?

07/18 02:38 PDT

### Testgrid link

https://testgrid.k8s.io/sig-release-master-blocking#build-master

### Reason for failure (if possible)

```
#5 [debbase 1/1] FROM registry.k8s.io/build-image/debian-base-arm64:bookworm-v1.0.0@sha256:e6cacb9e5c53a0dda096ecb3b0b3618cd761234d5e33a8902bf520348f8df6e4
#5 extracting sha256:a26c1c33c5b49f38510705cca6dd0faf7eed23b1af43bcdf3fe7852cf40214b6 4.1s done
#5 DONE 5.1s

#8 [stage-1  2/12] COPY --from=debbase /usr/bin/diff /usr/local/bin/
#8 CACHED

#9 [stage-1  3/12] COPY --from=debbase /lib/x86_64-linux-gnu/libc.so.6 /lib/x86_64-linux-gnu
#9 ERROR: failed to calculate checksum of ref 069374ab-5209-44e7-a102-6a18324c3aca::pmh36k4qsjwr5tb40qhdlnpl9: failed to walk /var/lib/docker/tmp/buildkit-mount316227079/usr/lib/x86_64-linux-gnu: lstat /var/lib/docker/tmp/buildkit-mount316227079/usr/lib/x86_64-linux-gnu: no such file or directory

#10 [stage-1  5/12] COPY --from=debbase /lib64/ld-linux-x86-64.so.2 /lib64
#10 ERROR: failed to calculate checksum of ref 069374ab-5209-44e7-a102-6a18324c3aca::pmh36k4qsjwr5tb40qhdlnpl9: failed to walk /var/lib/docker/tmp/buildkit-mount2116837088/lib64: lstat /var/lib/docker/tmp/buildkit-mount2116837088/lib64: no such file or directory

#11 [stage-1  4/12] COPY --from=debbase /lib/x86_64-linux-gnu/libpthread.so.0 /lib/x86_64-linux-gnu
#11 ERROR: failed to calculate checksum of ref 069374ab-5209-44e7-a102-6a18324c3aca::pmh36k4qsjwr5tb40qhdlnpl9: failed to walk /var/lib/docker/tmp/buildkit-mount953793106/usr/lib/x86_64-linux-gnu: lstat /var/lib/docker/tmp/buildkit-mount953793106/usr/lib/x86_64-linux-gnu: no such file or directory
------
 > [stage-1  3/12] COPY --from=debbase /lib/x86_64-linux-gnu/libc.so.6 /lib/x86_64-linux-gnu:
------
------
 > [stage-1  4/12] COPY --from=debbase /lib/x86_64-linux-gnu/libpthread.so.0 /lib/x86_64-linux-gnu:
------
------
 > [stage-1  5/12] COPY --from=debbase /lib64/ld-linux-x86-64.so.2 /lib64:
------
Dockerfile:26
--------------------
  24 |     COPY --from=debbase /lib/x86_64-linux-gnu/libc.so.6 /lib/x86_64-linux-gnu
  25 |     COPY --from=debbase /lib/x86_64-linux-gnu/libpthread.so.0 /lib/x86_64-linux-gnu
  26 | >>> COPY --from=debbase /lib64/ld-linux-x86-64.so.2 /lib64
  27 |     
  28 |     COPY cluster /kubernetes/cluster
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref 069374ab-5209-44e7-a102-6a18324c3aca::pmh36k4qsjwr5tb40qhdlnpl9: failed to walk /var/lib/docker/tmp/buildkit-mount2116837088/lib64: lstat /var/lib/docker/tmp/buildkit-mount2116837088/lib64: no such file or directory
make[1]: *** [Makefile:63: build] Error 1
```

### Anything else we need to know?

_No response_

### Relevant SIG(s)

/sig release

--- a/test/conformance/image/Dockerfile
+++ b/test/conformance/image/Dockerfile
@@ -12,18 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASEIMAGE
 ARG RUNNERIMAGE
-
-FROM ${BASEIMAGE} as debbase
-
 FROM ${RUNNERIMAGE}
-
-# This is a dependency for `kubectl diff` tests
-COPY --from=debbase /usr/bin/diff /usr/local/bin/
-COPY --from=debbase /lib/x86_64-linux-gnu/libc.so.6 /lib/x86_64-linux-gnu
-COPY --from=debbase /lib/x86_64-linux-gnu/libpthread.so.0 /lib/x86_64-linux-gnu
-COPY --from=debbase /lib64/ld-linux-x86-64.so.2 /lib64
 
 COPY cluster /kubernetes/cluster
 COPY ginkgo /usr/local/bin/

--- a/test/conformance/image/Makefile
+++ b/test/conformance/image/Makefile
@@ -34,12 +34,7 @@ CLUSTER_DIR?=$(shell pwd)/../../../cluster/
 # This is defined in root Makefile, but some build contexts do not refer to them
 KUBE_BASE_IMAGE_REGISTRY?=registry.k8s.io
 BASE_IMAGE_VERSION?=bookworm-v1.0.0
-BASEIMAGE?=${KUBE_BASE_IMAGE_REGISTRY}/build-image/debian-base-${ARCH}:${BASE_IMAGE_VERSION}
-
-# Keep debian releases (e.g. debian 11 == bullseye) consistent 
-# between BASE_IMAGE_VERSION and DISTROLESS_IMAGE images
-DISTROLESS_IMAGE?=base-debian11
-RUNNERIMAGE?=gcr.io/distroless/${DISTROLESS_IMAGE}:latest
+RUNNERIMAGE?=${KUBE_BASE_IMAGE_REGISTRY}/build-image/debian-base-${ARCH}:${BASE_IMAGE_VERSION}
 
 TEMP_DIR:=$(shell mktemp -d -t conformance-XXXXXX)
 
@@ -68,7 +63,6 @@ endif
 		--load \
 		--pull \
 		-t ${REGISTRY}/conformance-${ARCH}:${VERSION} \
-		--build-arg BASEIMAGE=$(BASEIMAGE) \
 		--build-arg RUNNERIMAGE=$(RUNNERIMAGE) \
 		${TEMP_DIR}
 	rm -rf "${TEMP_DIR}"

